ocs_ci/deployment/helpers/external_cluster_helpers.py

```diff
@@ -296,6 +297,21 @@ def update_permission_caps(self, user=None):
         out = self.run_exporter_script(params=params)
         logger.info(f"updated permissions for the user are set as {out}")
 
+    def upload_config_ini_file(self, params):
+        """
+        Upload config.ini file that is used for external cluster
+        exporter script --config-file param
+        Args:
+            params (str): Parameter to pass to exporter script
+        Returns:
```

Comment:
add extra free line before Returns, and also at the end of docstring is also worth to add empty line